src/renderer/shared/util/sendAsset.js

```diff
@@ -33,10 +33,10 @@ export default async function sendAsset({ net, asset, amount, receiver, address,
       net,
       address,
       script: createScript(asset, 'transfer', [
-        u.reverseHex(wallet.getScriptHashFromAddress(address)),
-        u.reverseHex(wallet.getScriptHashFromAddress(receiver)),
-        new u.Fixed8(amount).toReverseHex()
-      ], false),
+        address,
+        receiver,
+        new u.Fixed8(amount)
+      ], true),
```

Comment:
It defaults to true if it's not passed :) 
Did you also confirm that it results in the same behavior?
LGTM though - going to let @mhuggins go through the PR for final check ^^

Reply:
I couldn't make sure that it defaults to true looking at: https://github.com/trueinsider/client/blob/0ff2aab8dfd7b89b14d2ce40df516917c36cd641/src/renderer/shared/util/createScript.js#L5

I did confirm by making another successful transaction on mainnet: https://neotracker.io/tx/c7be1aa52aa4f36df4dc190ee8cff1caace80cffcce1ffb39bae23bfc33adeb7

Reply:
You can find it right [here](https://github.com/trueinsider/client/blob/0ff2aab8dfd7b89b14d2ce40df516917c36cd641/src/renderer/browser/actions/makeTestInvokeActions.js#L34). Maybe not the best place for it :P

Reply:
That said - [doInvoke](https://github.com/trueinsider/client/blob/0ff2aab8dfd7b89b14d2ce40df516917c36cd641/src/renderer/browser/actions/makeInvokeActions.js#L13) doesn't have it by default...
We might refactor this a little to make it consistent :P

Reply:
Of couse I maybe wrong but I believe this doesn't apply in current case because I'm calling `createScript` function directly, not through `createActions`. And this function doesn't have default value for `encodeArgs` argument.